Wasm/Bytes.lean

```diff
@@ -76,22 +83,30 @@ mutual
     -- TODO: handle locals
     | _ => sorry
 
-  partial def extractAdd (α : Type') : ByteArray :=
-    b $ match α with
-    | .i 32 => 0x6a
-    | .i 64 => 0x7c
-    | .f 32 => 0x92
-    | .f 64 => 0xa0
-
   partial def extractOp (x : Operation) : ByteArray :=
     match x with
     | .nop => b 0x01
     -- TODO: signed consts exist??? We should check the spec carefully.
-    | .const (.i _) (.i ci) => ByteArray.mk #[0x41] ++ sLeb128 ci.val
+    | .const (.i 32) (.i ci) => b 0x41 ++ sLeb128 ci.val
+    | .const (.i 64) (.i ci) => b 0x42 ++ sLeb128 ci.val
     | .const _ _ => sorry -- TODO: float binary encoding
     | .add t g1 g2 =>
       -- Enter stackman
       extractGet' g1 ++ extractGet' g2 ++ extractAdd t
+    | .block ts ops =>
+      let bts := flatten $ ts.map (b ∘ ttoi)
+      let obs := bts ++ b ops.length.toUInt8 ++ flatten (ops.map extractOp)
```

Comment:
Im 99% sure we need LEB everywhere. There can trivial ly be more than 255 instructions in else / then